include/deal.II/fe/fe_values.h

```diff
@@ -2633,7 +2633,7 @@ class FEValuesBase :
    * keep around between subsequent calls to reinit() in order to compute the
    * cell similarity.
    */
-  boost::signals2::connection tria_listener;
+  std::vector<boost::signals2::connection> tria_listener;
```

Comment:
Maybe two connection objects with different names make it more clear what is going on?